doc/source/dev/gitwash/development_workflow.rst

```diff
@@ -117,7 +84,7 @@ Overview
 
    # hack hack
    git add my_new_file
-   git commit -am 'ENH: some message'
+   git commit -a
```

Comment:
I agree with Chuck that you should not use `-a` after `git add` though - that's a very bad habit to get into.  I've often carefully done `git add` to select the changes I want and then thoughtlessly done `git commit -a` therefore wiping out all my work on selecting changes.